recipes/openscenegraph/all/conandata.yml

```diff
@@ -16,3 +16,11 @@ patches:
       base_path: source_subfolder
     - patch_file: patches/0006-Declare-result-as-LONG-for-Mingw-build.patch
       base_path: source_subfolder
+    - patch_file: patches/0007-fix-msvc-with-std-c++17.patch
+      base_path: source_subfolder
```

Comment:
Could please add patch_source and patch type entries: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patches-fields

So we will know that's not an isolated patch created only for Conan packages.